pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -124,6 +125,10 @@ func InitializeSecurityPolicy(service common.Service) (*SecurityPolicyService, e
 }
 
 func (service *SecurityPolicyService) CreateOrUpdateSecurityPolicy(obj interface{}) error {
+	if !nsxutil.IsLicensed(nsxutil.FeatureDFW) {
```

Comment:
if DFW license is disabled,  from commit msg, it seems we only handle sp delete request, maybe we should block all create/update operation in reconciler instead

Reply:
CreateOrUpdateSecurityPolicy is the  only entry for securitypolicy, blocking here is the same in the reconciler